### DISCUSSION TITLE
Problema con i messaggi audio

### DISCUSSION THREAD

**Discussion by rappox** [globaleaks/globaleaks-whistleblowing-software#4356] [Q&A] [answered]
Buongiorno sto provando a configurare la piattaforma per poter utilizzare anche segnalazioni audio.
Vado in question templates: setto question: invia audio; type:voce
poi vado in questionnaires creo un duplicato dell'originale, lo edito, edito il primo step,add question from template: invia audio e salvo.
provo a creare una prima segnalazione registro una prima segnalazione e sembra tutto ok, poi se non la invio ma aggiorno semplicemente la pagina e creo nuovamente una segnalazione, provando a registrare però non mi registra più si blocca subito.
sbaglio qualcosa nella configurazione? 
Grazie

**Comment by evilaliv3** [comment]
Buongiorno @rappox 

che del software stai usando? hai un link che possiamo provare?

su che browser/versione stati effettuando la tua prova?

**Reply by rappox** [reply]
Ciao, ho provato anche la versione try https://fastesting.try.globaleaks.org/ e mi da lo stesso problema
da android: chrome 131.0.6778.135
da windows chrome 131.0.6778.140

**Reply by evilaliv3** [reply]
Grazie, riproviamo e vi aggiorniamo.

**Reply by rappox** [reply]
Grazie!

**Comment by evilaliv3** [comment, marked as answer]
Grazie @rappox, problema identificato.

La problematica è dovuta ad un bug del software e si verifica quando la domanda vocale è impostata usando i template di domanda e si verifica solo su chrome che implementa correttamente l'header Permission Policy con cui blocchiamo il microfono.

Non si verifica invece quando la domanda è creata direttamente sul questionario. Pui verificare questo comportamento su: https://fastesting.try.globaleaks.org/

**Reply by rappox** [reply]
Perfetto grazie mille!
